buildSrc/src/main/java/Repositories.kt

```diff
@@ -30,10 +30,18 @@ fun ScriptHandler.addDefaultInputRepository() {
 
 internal fun RepositoryHandler.addDefaultInputRepository() {
   google()
-  jcenter()
   maven {
     setUrl("https://jitpack.io")
   }
+  jcenter {
```

Comment:
Same than in other `build.gradle.kts`: could you move the `mavenCentral` up and leave the `jcenter `where it was before ?

Reply:
done